src/controller/mailing_controller.py

```diff
@@ -0,0 +1,61 @@
+from smtplib import SMTPResponseException
+from flask import request, Blueprint
+from flask import current_app as app
+from http import HTTPStatus
+
+from src.service.mailing.mailservice import (CreateReservationConfirmationMessage, CreateInvoiceMessage,
+                                             CreateResetPasswordMessage, CreatePaymentConfirmationMessage,
+                                             CreateActivationMessage, MailService)
+
+mailing = Blueprint('mailing', __name__, url_prefix='/mailing')
+
+def SelectMessageCreator(message_type, data_id, recipients):
+    if message_type == 1:
+        return CreateActivationMessage(data_id, recipients).create_message()
+
+    if message_type == 2:
+        return CreateResetPasswordMessage(data_id, recipients).create_message()
+
+    if message_type == 3:
+      return CreateReservationConfirmationMessage(data_id, recipients).create_message()
+
+    if message_type == 4:
+      return CreateInvoiceMessage(data_id, recipients).create_message()
+
+    if message_type == 5:
+        return CreatePaymentConfirmationMessage(data_id, recipients).create_message()
+
+@mailing.route('/sendmail', methods=['POST'])
+def sendmail():
+    data_id = request.args.get('data_id', type=int)
+    address = request.args.get('address', type=str, default='')
+    message_type = request.args.get('message_type', type=int)
+
+    if data_id < 0  or not address or message_type not in range(1,6):

```

Comment:
Rozdzielamy tego ifa na 3 i w każdym logujemy co jest nie tak + zwracamy jaki parametr jest nie tak. 
Dodatkowo zrób enumy, bo nie wiem dlaczego range(1, 6) jest dobry, nic mi to nie mówi